core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -274,6 +287,43 @@ public List<ActiveScan> getActiveScans(String tserver)
     }
   }
 
+  @Override
+  public List<ActiveScan> getActiveScans(ServerId server)
+      throws AccumuloException, AccumuloSecurityException {
+
+    @SuppressWarnings("unused")
+    var unused = Objects.nonNull(server);
```

Comment:
This isn't correct. This returns a boolean. What you want is:

```suggestion
    Objects.requireNonNull(server);
```

Reply:
Thanks, fixed in de7df88